install/configuring_inventory_file.adoc

```diff
@@ -455,12 +455,13 @@ Groups and Host Mappings] for the current method of setting node labels.
 a|This variable configures additional `docker` options in
 *_/etc/sysconfig/docker_*, such as options used in
 xref:host_preparation.adoc#managing-docker-container-logs[Managing Container Logs].
-It is recommended to use `json-file`.
+We recommend that you use `json-file`.
 
 The following example shows the configuration of Docker to use the `json-file` log
-driver, where Docker will rotate between three 1 MB log files:
+driver, where Docker can rotate between three 1 MB log files, and signature verification is not required.
+When supplying additional options, be sure you maintain the single quotation mark formatting:
 ----
-"--log-driver json-file --log-opt max-size=1M --log-opt max-file=3"
+OPTIONS=' --selinux-enabled --log-opt  max-size=1M --log-opt max-file=3 --insecure-registry 172.30.0.0/16 --log-driver=json-file --signature-verification=False'
```

Comment:
@anpingli - does it matter whether the variable is "False" instead of "false"?

Reply:
Either is OK, but I changed to the default, which is lowercase.